docs/05-technical-reference/00-architecture/assets/evnt-architecture.svg

```diff
@@ -1,3 +1,3 @@
px" text-anchor="middle">watches subscription</text></switch></g><g transform="translate(-0.5 -0.5)"><switch><foreignObject style="overflow: visible; text-align: left;" pointer-events="none" width="100%" height="100%" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility"><div xmlns="http://www.w3.org/1999/xhtml" style="display: flex; align-items: unsafe center; justify-content: unsafe center; width: 1px; height: 1px; padding-top: 126px; margin-left: 895px;"><div style="box-sizing: border-box; font-size: 0; text-align: center; "><div style="display: inline-block; font-size: 12px; font-family: Helvetica; color: #000000; line-height: 1.2; pointer-events: none; white-space: nowrap; ">dispatches event</div></div></div></foreignObject><text x="895" y="130" fill="#000000" font-family="Helvetica" font-size="12px" text-anchor="middle">dispatches event</text></switch></g><path d="M 360 196 L 400 196 L 433.63 195.58" fill="none" stroke="#959ca4" stroke-miterlimit="10" pointer-events="none"/><path d="M 438.88 195.51 L 431.93 199.1 L 433.63 195.58 L 431.84 192.1 Z" fill="#959ca4" stroke="#959ca4" stroke-miterlimit="10" pointer-events="none"/><path d="M 681.26 210.97 L 623.73 211 L 570.18 210.9" fill="none" stroke="#959ca4" stroke-miterlimit="10" pointer-events="none"/><path d="M 564.93 210.89 L 571.94 207.41 L 570.18 210.9 L 571.93 214.41 Z" fill="#959ca4" stroke="#959ca4" stroke-miterlimit="10" pointer-events="none"/><path d="M 571 191 L 673.63 191" fill="none" stroke="#959ca4" stroke-miterlimit="10" pointer-events="none"/><path d="M 678.88 191 L 671.88 194.5 L 673.63 191 L 671.88 187.5 Z" fill="#959ca4" stroke="#959ca4" stroke-miterlimit="10" pointer-events="none"/><ellipse cx="890" cy="96.05" rx="20" ry="20" fill="#0a6ed1" stroke="#0a6ed1" pointer-events="none"/><g transform="translate(-0.5 -0.5)"><switch><foreignObject style="overflow: visible; text-align: left;" pointer-events="none" width="100%" height="100%" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility"><div xmlns="http://www.w3.org/1999/xhtml" style="display: flex; align-items: unsafe center; justify-content: unsafe center; width: 38px; height: 1px; padding-top: 96px; margin-left: 871px;"><div style="box-sizing: border-box; font-size: 0; text-align: center; "><div style="display: inline-block; font-size: 12px; font-family: Helvetica; color: #000000; line-height: 1.2; pointer-events: none; white-space: normal; word-wrap: normal; "><font color="#ffffff"><span style="font-size: 19px">6</span></font></div></div></div></foreignObject><text x="890" y="100" fill="#000000" font-family="Helvetica" font-size="12px" text-anchor="middle">6</text></switch></g></g><switch><g requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility"/><a transform="translate(0,-5)" xlink:href="https://www.diagrams.net/doc/faq/svg-export-text-problems" target="_blank"><text text-anchor="middle" font-size="10px" x="50%" y="100%">Viewer does not support full SVG 1.1</text></a></switch></svg>
```

Comment:
```suggestion
<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" version="1.1" width="1401px" height="512px" viewBox="-0.5 -0.5 1401 512" content="&lt;mxfile host=&quot;app.diagrams.net&quot; modified=&quot;2021-06-22T14:26:44.771Z&quot; agent=&quot;5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/91.0.4472.114 Safari/537.36&quot; etag=&quot;c3REqhju8n1bDMwUmbKp&quot; version=&quot;14.8.0&quot; type=&quot;device&quot;&gt;&lt;diagram id=&quot;36Z0lrfrI7yJ4n3zALGV&quot; name=&quot;Page-1&quot;&gt;5Vxfc+MoEv80rrp7iEv/kKXHJI5zVTezmdvs3WX2TbawrYssPBJO4vn0BwgkBMhWbNmJZ71bu6JBDXT/aLoblIF7u3q7z6P18iuKYTpwrPht4I4HjuMAe0T+RynbkhL4nLDIk7gk2TXhMfkJOdHi1E0Sw6LRECOU4mTdJM5QlsEZbtCiPEevzWZzlDZ7XUcLqBEeZ1GqU/+bxHjJqbZl1RX/gMliybsOAK+YRrPnRY42Ge9v4Lhz9iurV5HgxdsXyyhGrxLJvRu4tzlCuHxavd3ClMpWiK18b9JSW407hxnu8sIc/Pl07YUPAP94wJOnMF08/OvKBSWblyjdQDEPNlq8FRJic4SUizVwb16XCYaP62hGa18JJghtiVcpKdnkscA5eoa3KEU5oWQoI81u9KHy0b/AHMM3icSHfg/RCuJ8S5rw2ishxldJSZ5Q0lJSEBDIijgyFhWzWjrkgQvILKxgv1io7tfVjAV2yBRu6KwSgrDrNFlkhIYRayf0T6VYy5QKbZ6kqRAZgdHEmviTybvkVq2PaCrGZ+2Upz3iUto2l6MsXjswiNdzehCvc4x4d8pufDdxxuPTyi50mrIb6bIzItNxexCdaxCdn2IqCURmKsvQ/7FBouKqYJb3mjSwvfVbXUmeFvz/jMs0VynH8b1er1OyGHCCMtLmtpQ0ygVzMt2Sf7PPFnI/UzyaS7SiyMymxXqn4FomQRseOwWN9fE2Wywxw6Li+5pu2VZJHNMu2eD53m572jZAWPjsd+ReIGyVYcHZtmHBjXpYb97x6w3sXG+C8O8CUmD8Fq1gwVRVq3r6XvUTQeIO+7KsZk6KuGZnRD1kPDtUbgJVE3YyKEA/boBtue4QNK1vOAReAFwArJHl+cIflbExcnVshEMPyP+MjoeKyZcq9UUUmnWCimOCyuNmWszyZC1s6O8SDErOBxkYY1/9jLhIsueyfr7JZmzcrIRhgfcMvnPHe3exHmxsc4DnsbFxkkMusnGBNrT9mcyrqLWB4hv6YGgBfVk5YBiOTmN1/f797w93Gx1rr9vo+4ZdzO3D4x6daxt73q6iq2JbYLg63W7WfZfKIZkB1xBdpmuUZJjJEdwMANM4YbfBqODL6zT7lqeGDN6QbFzST4OCY4wgekBCuA8Js2pZ1Mp2LfaTSR0Rw9TcDpojuPyTII0ON90UFAQXBjDZ+DhjMA5PA7yR3fSWTGkUk+vcB9Jsex/UPjSQ++36j0cq3s00TYolc77XOXrbXlr4lkZTmN5U2UjFoTc4Hud0JhxP8SVGjtmXMGDQ93vAoO44wHgBH3kR5XiJFiiL0ruaqrgKdZsviHoSjPg/iPGWS5Gu7qZK4FuCn6Tn71STQ8BL4zeuWFbYikJGpvYkF6S3aLF+jZXEe23G5AC1Enczn3Ep8ZAJR/kCila8GRXgTt3nMI1w8tLMsJu0yF/9Rm2m5IC6Tcw4AjCCRTlO/pac8lYYOWE4FFtrxcsj4aK08Sqsy/lqrBnMqhl2Q14PmYMddsuQdGqkpQqYvzR2xZ0Gp1hGa/o426YJgX6+3+pMyzXyZVoRKgv0sMGEi9jscoQjHs6EFK9LlCc/yViidKCnCU5ujzw17+3ZQ90c+YFujWynhxSB3Z4j6AwKY8T9je9iBV2fL5C6IJ/C+VE8nx40aCsejR3YZ3Rp9MC0gFlcSf3ipev61l7pmo6FehGuHqTGCYke8ayJ6wsXMVBNUItLdDIMmw4390eA4nD7XRFgaLJVdlfTRMadrIu2ZHNrdpuGq9f+nSnPU9F70GOg6LFaF41zQF2HXh86PCyK70+HTqsO96Rx+bgo53wxjf5GhEH+JRO2jE9/p49soGxY82iVpNvy9RXKEM8tyU3qkVt04LyKAeiqbM4qM8RgZAjRgHrbBRAdUCq7q1GVhE4A0wqhjOkzHTigygBElfva2lVbgcSD2Dg1mxIPVQ39D9UHIXBcAIEM0aacP21ASxVC1AquSf2tWuakdM2alohROVDx0mdbHhSboBiZaFFPp9lpZQ9oI8ki0PpSmYBbBUqxWVG2DIKPsAJs+JV9MNVKgi3NQhfVVVWV1mrbAaj1qFsGoGazlRQqqZ9bk6rOk6q4VTHWEetS0RdS/yqmWLEClkxswp2309ZFt7OSz23JQQg+zJKLHf1oS37gMZrRunt7Dukue+N2A2XjdoznWyfT+MekpKr00jAMg2aKyXP25JhY6RvMEzJZ6lAzYhyReDOu2MfX9DYm9brTqCiSWUmcJGmrrvfH9XL2iSvo1MmnkZKwdDwlC9mSfCKzj7ZSMx5JdO+HZzvbcll2lZIzv0AeyiEcmq862c2ynVcAOmWlMoQ72JK2BFQc5c8P5K0EMzjTpa5fuWQ/Ql/kUZzAOqMqsujlsjp7uirseBRvslUkrFSSmwdZK+9jrVXDUjXMlB92s1MdTJPI14uk/HfpWeqrka4nBbWngtgrLPqqzl8IjXdlHQYXOS0vzKBsGcXK7c80dgZHD+nMXWdzLOOTZAvmBRHdojR9f077r2s9/EC1Hn5n6xH04emYrvF8tG/r/tK+rXp368y+rfsZoxnwS2tcDV5tx7jIT6ZxR9P4LIcRZhn7JJvnEZHKZoY3OdQEe3GZe3VxeXqeoDq87jtt7+r++Wt1NFI0rghfuJgD9fzfMzu9J5O0fnKvn0NdvpTVmznmneJkx1AjNy7wo/v1/sf9fyZvy5/fv27urtxAt74fFm3svXpjDDcO8OhOndPw1FyD5Q3lOzCh1+TYcglG46seFne9qPPeXIlnmfvpK/VhhKKQ2Xmh+AnRA3wlO66u/q5w8UfhbkYtcOlNo/bn0miVdRh6jidnHoaWJbISLXcFDcmHrvho5hPKW2q9Y0aLPMU1vPdiRr0sUV2LPxdm9mZFPyCg8X/pgEa9VxFQv8Sqf/qFpFPFNovi6/QPf729h+HTfBuvHl+8L1eeHvBoAm4ajUP+ZELbnwSQvw6DJLQaNC5YXrEbllE+k1NMLfI3aKn9xEwNfAwHpL43BIbvLN2A0INT6aKHM4tunzX9DueQSItdNnmG5Q2US/8+dzeWrH6go3715pFgbhTUrufI05Bk2wYc9RFymDHUfk87Tl7MhrukjcsD0OpDQJQXbOdck3CORYjy34hhH5fMYFHAYr/1bnTcO3ZSOMfHIacPXNTHUwIZfhMZoa8hw7V8g4VxT4WMDn8cx3i4JOmmcUiuf4uzz93XfLcOApbkBQwbo6Ad6eE5Yh8W2gsc47p+9zcbvvMuvgd/sDHgN5yk5vXdJvfu/w==&lt;/diagram&gt;&lt;/mxfile&gt;" style="background-color: rgb(255, 255, 255);"><defs/><g><rect x="0" y="0" width="1400" height="510" fill="#ffffff" stroke="none" pointer-events="all"/><rect x="190" y="10" width="1180" height="420" rx="63" ry="63" fill="#f0f6ff" stroke="none" pointer-events="all"/><rect x="940" y="70" width="400" height="230" rx="34.5" ry="34.5" fill="#def2dd" stroke="none" pointer-events="all"/><rect x="20" y="170" width="110" height="70" fill="#ffffff" stroke="#666666" pointer-events="all"/><g transform="translate(-0.5 -0.5)"><switch><foreignObject style="overflow: visible; text-align: left;" pointer-events="none" width="100%" height="100%" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility"><div xmlns="http://www.w3.org/1999/xhtml" style="display: flex; align-items: unsafe center; justify-content: unsafe center; width: 108px; height: 1px; padding-top: 205px; margin-left: 21px;"><div style="box-sizing: border-box; font-size: 0; text-align: center; "><div style="display: inline-block; font-size: 14px; font-family: Helvetica; color: #000000; line-height: 1.2; pointer-events: all; white-space: normal; word-wrap: normal; "><font style="font-size: 14px"><br /><font style="font-size: 14px">Application Connector</font></font><font style="font-size: 14px"><font style="font-size: 14px"> <br /></font><br style="font-size: 14px" /></font></div></div></div></foreignObject><text x="75" y="209" fill="#000000" font-family="Helvetica" font-size="14px" text-anchor="middle">Application Conn...</text></switch></g><rect x="1053.5" y="79.55" width="173" height="9.45" fill="none" stroke="none" pointer-events="all"/><g transform="translate(-0.5 -0.5)"><switch><foreignObject style="overflow: visible; text-align: left;" pointer-events="none" width="100%" height="100%" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility"><div xmlns="http://www.w3.org/1999/xhtml" style="display: flex; align-items: unsafe center; justify-content: unsafe center; width: 171px; height: 1px; padding-top: 84px; margin-left: 1055px;"><div style="box-sizing: border-box; font-size: 0; text-align: center; "><div style="display: inline-block; font-size: 15px; font-family: Helvetica; color: #000000; line-height: 1.2; pointer-events: all; white-space: normal; word-wrap: normal; "><font style="font-size: 15px"><b>User Namespace</b></font></div></div></div></foreignObject><text x="1140" y="89" fill="#000000" font-family="Helvetica" font-size="15px" text-anchor="middle">User Namespace</text></switch></g><rect x="1197.99" y="137.06" width="70" height="125.97" fill="#ffffff" stroke="#666666" transform="rotate(90,1232.99,200.05)" pointer-events="all"/><g transform="translate(-0.5 -0.5)"><switch><foreignObject style="overflow: visible; text-align: left;" pointer-events="none" width="100%" height="100%" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility"><div xmlns="http://www.w3.org/1999/xhtml" style="display: flex; align-items: unsafe center; justify-content: unsafe center; width: 124px; height: 1px; padding-top: 200px; margin-left: 1171px;"><div style="box-sizing: border-box; font-size: 0; text-align: center; "><div style="display: inline-block; font-size: 14px; font-family: Helvetica; color: #000000; line-height: 1.2; pointer-events: all; white-space: normal; word-wrap: normal; "><span style="font-size: 12px">Subscription CR</span><br style="font-size: 12px" /><span style="font-size: 12px">sink: function: test</span><span style="font-size: 14px"><br style="font-size: 14px" /></span></div></div></div></foreignObject><text x="1233" y="204" fill="#000000" font-family="Helvetica" font-size="14px" text-anchor="middle">Subscription CR...</text></switch></g><rect x="220" y="70" width="660" height="320" rx="48" ry="48" fill="#def2dd" stroke="none" pointer-events="all"/><rect x="440" y="74.28" width="200" height="20" fill="none" stroke="none" pointer-events="all"/><g transform="translate(-0.5 -0.5)"><switch><foreignObject style="overflow: visible; text-align: left;" pointer-events="none" width="100%" height="100%" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility"><div xmlns="http://www.w3.org/1999/xhtml" style="display: flex; align-items: unsafe center; justify-content: unsafe center; width: 1px; height: 1px; padding-top: 84px; margin-left: 540px;"><div style="box-sizing: border-box; font-size: 0; text-align: center; "><div style="display: inline-block; font-size: 15px; font-family: Helvetica; color: #000000; line-height: 1.2; pointer-events: all; white-space: nowrap; "><font style="font-size: 15px"><b>kyma-system Namespace</b></font></div></div></div></foreignObject><text x="540" y="89" fill="#000000" font-family="Helvetica" font-size="15px" text-anchor="middle">kyma-system Namespace</text></switch></g><rect x="735" y="20" width="110" height="20" fill="none" stroke="none" pointer-events="all"/><g transform="translate(-0.5 -0.5)"><switch><foreignObject style="overflow: visible; text-align: left;" pointer-events="none" width="100%" height="100%" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility"><div xmlns="http://www.w3.org/1999/xhtml" style="display: flex; align-items: unsafe center; justify-content: unsafe center; width: 1px; height: 1px; padding-top: 30px; margin-left: 790px;"><div style="box-sizing: border-box; font-size: 0; text-align: center; "><div style="display: inline-block; font-size: 15px; font-family: Helvetica; color: #D2D5D9; line-height: 1.2; pointer-events: all; white-space: nowrap; "><font color="#000000" style="font-size: 15px"><b style="font-size: 15px">Kyma cluster</b></font></div></div></div></foreignObject><text x="790" y="34" fill="#D2D5D9" font-family="Helvetica" font-size="15px" text-anchor="middle">Kyma cluster</text></switch></g><rect x="260" y="172.05" width="120" height="66" fill="#ffffff" stroke="#666666" pointer-events="all"/><g transform="translate(-0.5 -0.5)"><switch><foreignObject style="overflow: visible; text-align: left;" pointer-events="none" width="100%" height="100%" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility"><div xmlns="http://www.w3.org/1999/xhtml" style="display: flex; align-items: unsafe center; justify-content: unsafe center; width: 118px; height: 1px; padding-top: 205px; margin-left: 261px;"><div style="box-sizing: border-box; font-size: 0; text-align: center; "><div style="display: inline-block; font-size: 14px; font-family: Helvetica; color: #000000; line-height: 1.2; pointer-events: all; white-space: normal; word-wrap: normal; "><font style="font-size: 14px"><font style="font-size: 14px">NATS publisher proxy</font><br style="font-size: 14px" /></font></div></div></div></foreignObject><text x="320" y="209" fill="#000000" font-family="Helvetica" font-size="14px" text-anchor="middle">NATS publisher pr...</text></switch></g><path d="M 130 205 L 185 205 Q 195 205 205 205.01 L 253.63 205.05" fill="none" stroke="#666666" stroke-miterlimit="10" pointer-events="stroke"/><path d="M 258.88 205.05 L 251.88 208.54 L 253.63 205.05 L 251.88 201.54 Z" fill="#666666" stroke="#666666" stroke-miterlimit="10" pointer-events="all"/><path d="M 490 166.5 C 490 133.17 558 133.17 558 166.5 L 558 243.5 C 558 276.83 490 276.83 490 243.5 Z" fill="#ffffff" stroke="#666666" stroke-miterlimit="10" transform="rotate(90,524,205)" pointer-events="all"/><path d="M 490 166.5 C 490 191.5 558 191.5 558 166.5" fill="none" stroke="#666666" stroke-miterlimit="10" transform="rotate(90,524,205)" pointer-events="all"/><g transform="translate(-0.5 -0.5)"><switch><foreignObject style="overflow: visible; text-align: left;" pointer-events="none" width="100%" height="100%" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility"><div xmlns="http://www.w3.org/1999/xhtml" style="display: flex; align-items: unsafe center; justify-content: unsafe center; width: 87px; height: 1px; padding-top: 205px; margin-left: 462px;"><div style="box-sizing: border-box; font-size: 0; text-align: center; "><div style="display: inline-block; font-size: 14px; font-family: Helvetica; color: #000000; line-height: 1.2; pointer-events: all; white-space: normal; word-wrap: normal; "><font style="font-size: 14px">NATS<br /> server</font></div></div></div></foreignObject><text x="505" y="209" fill="#000000" font-family="Helvetica" font-size="14px" text-anchor="middle">NATS...</text></switch></g><rect x="135" y="181" width="110" height="20" fill="none" stroke="none" pointer-events="all"/><g transform="translate(-0.5 -0.5)"><switch><foreignObject style="overflow: visible; text-align: left;" pointer-events="none" width="100%" height="100%" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility"><div xmlns="http://www.w3.org/1999/xhtml" style="display: flex; align-items: unsafe center; justify-content: unsafe center; width: 1px; height: 1px; padding-top: 191px; margin-left: 190px;"><div style="box-sizing: border-box; font-size: 0; text-align: center; "><div style="display: inline-block; font-size: 12px; font-family: Helvetica; color: #000000; line-height: 1.2; pointer-events: all; white-space: nowrap; "><font style="font-size: 12px">Publishes events</font></div></div></div></foreignObject><text x="190" y="195" fill="#000000" font-family="Helvetica" font-size="12px" text-anchor="middle">Publishes events</text></switch></g><rect x="380" y="181" width="80" height="20" fill="none" stroke="none" pointer-events="all"/><g transform="translate(-0.5 -0.5)"><switch><foreignObject style="overflow: visible; text-align: left;" pointer-events="none" width="100%" height="100%" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility"><div xmlns="http://www.w3.org/1999/xhtml" style="display: flex; align-items: unsafe center; justify-content: unsafe center; width: 1px; height: 1px; padding-top: 191px; margin-left: 420px;"><div style="box-sizing: border-box; font-size: 0; text-align: center; "><div style="display: inline-block; font-size: 12px; font-family: Helvetica; color: #000000; line-height: 1.2; pointer-events: all; white-space: nowrap; ">sends event</div></div></div></foreignObject><text x="420" y="195" fill="#000000" font-family="Helvetica" font-size="12px" text-anchor="middle">sends event</text></switch></g><rect x="590" y="172.05" width="110" height="20" fill="none" stroke="none" pointer-events="all"/><g transform="translate(-0.5 -0.5)"><switch><foreignObject style="overflow: visible; text-align: left;" pointer-events="none" width="100%" height="100%" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility"><div xmlns="http://www.w3.org/1999/xhtml" style="display: flex; align-items: unsafe center; justify-content: unsafe center; width: 1px; height: 1px; padding-top: 182px; margin-left: 645px;"><div style="box-sizing: border-box; font-size: 0; text-align: center; "><div style="display: inline-block; font-size: 12px; font-family: Helvetica; color: #000000; line-height: 1.2; pointer-events: all; white-space: nowrap; ">dispatches events</div></div></div></foreignObject><text x="645" y="186" fill="#000000" font-family="Helvetica" font-size="12px" text-anchor="middle">dispatches events</text></switch></g><ellipse cx="910" cy="300" rx="20" ry="20" fill="#0a6ed1" stroke="#0a6ed1" pointer-events="all"/><g transform="translate(-0.5 -0.5)"><switch><foreignObject style="overflow: visible; text-align: left;" pointer-events="none" width="100%" height="100%" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility"><div xmlns="http://www.w3.org/1999/xhtml" style="display: flex; align-items: unsafe center; justify-content: unsafe center; width: 38px; height: 1px; padding-top: 300px; margin-left: 891px;"><div style="box-sizing: border-box; font-size: 0; text-align: center; "><div style="display: inline-block; font-size: 12px; font-family: Helvetica; color: #000000; line-height: 1.2; pointer-events: all; white-space: normal; word-wrap: normal; "><font color="#ffffff" style="font-size: 19px">1</font></div></div></div></foreignObject><text x="910" y="304" fill="#000000" font-family="Helvetica" font-size="12px" text-anchor="middle">1</text></switch></g><ellipse cx="635" cy="300" rx="20" ry="20" fill="#0a6ed1" stroke="#0a6ed1" pointer-events="all"/><g transform="translate(-0.5 -0.5)"><switch><foreignObject style="overflow: visible; text-align: left;" pointer-events="none" width="100%" height="100%" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility"><div xmlns="http://www.w3.org/1999/xhtml" style="display: flex; align-items: unsafe center; justify-content: unsafe center; width: 38px; height: 1px; padding-top: 300px; margin-left: 616px;"><div style="box-sizing: border-box; font-size: 0; text-align: center; "><div style="display: inline-block; font-size: 12px; font-family: Helvetica; color: #000000; line-height: 1.2; pointer-events: all; white-space: normal; word-wrap: normal; "><font color="#ffffff" style="font-size: 19px">2</font><span style="color: rgba(0 , 0 , 0 , 0) ; font-family: monospace ; font-size: 0px ; white-space: nowrap">%3CmxGraphModel%3E%3Croot%3E%3CmxCell%20id%3D%220%22%2F%3E%3CmxCell%20id%3D%221%22%20parent%3D%220%22%2F%3E%3CmxCell%20id%3D%222%22%20value%3D%22%26lt%3Bfont%20color%3D%26quot%3B%23ffffff%26quot%3B%20style%3D%26quot%3Bfont-size%3A%2019px%26quot%3B%26gt%3B1%26lt%3B%2Ffont%26gt%3B%22%20style%3D%22ellipse%3BwhiteSpace%3Dwrap%3Bhtml%3D1%3BstrokeColor%3D%230A6ED1%3BfillColor%3D%230A6ED1%3B%22%20vertex%3D%221%22%20parent%3D%221%22%3E%3CmxGeometry%20x%3D%22185%22%20y%3D%22220%22%20width%3D%2240%22%20height%3D%2240%22%20as%3D%22geometry%22%2F%3E%3C%2FmxCell%3E%3C%2Froot%3E%3C%2FmxGraphModel%3E</span></div></div></div></foreignObject><text x="635" y="304" fill="#000000" font-family="Helvetica" font-size="12px" text-anchor="middle">2%3CmxG...</text></switch></g><ellipse cx="420" cy="245.05" rx="20" ry="20" fill="#0a6ed1" stroke="#0a6ed1" pointer-events="all"/><g transform="translate(-0.5 -0.5)"><switch><foreignObject style="overflow: visible; text-align: left;" pointer-events="none" width="100%" height="100%" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility"><div xmlns="http://www.w3.org/1999/xhtml" style="display: flex; align-items: unsafe center; justify-content: unsafe center; width: 38px; height: 1px; padding-top: 245px; margin-left: 401px;"><div style="box-sizing: border-box; font-size: 0; text-align: center; "><div style="display: inline-block; font-size: 12px; font-family: Helvetica; color: #000000; line-height: 1.2; pointer-events: all; white-space: normal; word-wrap: normal; "><font color="#ffffff"><span style="font-size: 19px">4</span></font></div></div></div></foreignObject><text x="420" y="249" fill="#000000" font-family="Helvetica" font-size="12px" text-anchor="middle">4</text></switch></g><path d="M 760 246 L 760 256 Q 760 266 770 266 L 1233.17 266 Q 1243.17 266 1243.13 256 L 1243.08 241.28" fill="none" stroke="#666666" stroke-miterlimit="10" stroke-dasharray="3 3" pointer-events="stroke"/><path d="M 1243.07 236.03 L 1246.59 243.02 L 1243.08 241.28 L 1239.59 243.04 Z" fill="#666666" stroke="#666666" stroke-miterlimit="10" pointer-events="all"/><path d="M 970 165.05 L 1110 165.05 L 1110 165.05 L 1110 237.55 L 970 237.55 L 970 165.05 Z" fill="#ffffff" stroke="#666666" stroke-miterlimit="10" pointer-events="all"/><path d="M 1110 165.05 L 1110 165.05 L 1110 165.05 Z" fill-opacity="0.05" fill="#000000" stroke="none" pointer-events="all"/><path d="M 1110 165.05 L 1110 165.05 L 1110 165.05" fill="none" stroke="#666666" stroke-miterlimit="10" pointer-events="all"/><g transform="translate(-0.5 -0.5)"><switch><foreignObject style="overflow: visible; text-align: left;" pointer-events="none" width="100%" height="100%" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility"><div xmlns="http://www.w3.org/1999/xhtml" style="display: flex; align-items: unsafe center; justify-content: unsafe center; width: 138px; height: 1px; padding-top: 201px; margin-left: 971px;"><div style="box-sizing: border-box; font-size: 0; text-align: center; "><div style="display: inline-block; font-size: 14px; font-family: Helvetica; color: #000000; line-height: 1.2; pointer-events: all; white-space: normal; word-wrap: normal; "><font style="font-size: 14px">function: test</font></div></div></div></foreignObject><text x="1040" y="205" fill="#000000" font-family="Helvetica" font-size="14px" text-anchor="middle">function: test</text></switch></g><path d="M 769 166.05 L 769 155.17 Q 769 145.17 779 145.17 L 1029 145.17 Q 1039 145.17 1039 151.92 L 1039 158.68" fill="none" stroke="#666666" stroke-miterlimit="10" pointer-events="stroke"/><path d="M 1039 163.93 L 1035.5 156.93 L 1039 158.68 L 1042.5 156.93 Z" fill="#666666" stroke="#666666" stroke-miterlimit="10" pointer-events="all"/><path d="M 700 166.05 L 840 166.05 L 840 166.05 L 840 246.05 L 700 246.05 L 700 166.05 Z" fill="#ffffff" stroke="#666666" stroke-miterlimit="10" pointer-events="all"/><path d="M 840 166.05 L 840 166.05 L 840 166.05 Z" fill-opacity="0.05" fill="#000000" stroke="none" pointer-events="all"/><path d="M 840 166.05 L 840 166.05 L 840 166.05" fill="none" stroke="#666666" stroke-miterlimit="10" pointer-events="all"/><g transform="translate(-0.5 -0.5)"><switch><foreignObject style="overflow: visible; text-align: left;" pointer-events="none" width="100%" height="100%" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility"><div xmlns="http://www.w3.org/1999/xhtml" style="display: flex; align-items: unsafe center; justify-content: unsafe center; width: 138px; height: 1px; padding-top: 206px; margin-left: 701px;"><div style="box-sizing: border-box; font-size: 0; text-align: center; "><div style="display: inline-block; font-size: 14px; font-family: Helvetica; color: #000000; line-height: 1.2; pointer-events: all; white-space: normal; word-wrap: normal; "><font style="font-size: 14px">NATS eventing controller</font></div></div></div></foreignObject><text x="770" y="210" fill="#000000" font-family="Helvetica" font-size="14px" text-anchor="middle">NATS eventing contro...</text></switch></g><ellipse cx="190" cy="245.05" rx="20" ry="20" fill="#0a6ed1" stroke="#0a6ed1" pointer-events="all"/><g transform="translate(-0.5 -0.5)"><switch><foreignObject style="overflow: visible; text-align: left;" pointer-events="none" width="100%" height="100%" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility"><div xmlns="http://www.w3.org/1999/xhtml" style="display: flex; align-items: unsafe center; justify-content: unsafe center; width: 38px; height: 1px; padding-top: 245px; margin-left: 171px;"><div style="box-sizing: border-box; font-size: 0; text-align: center; "><div style="display: inline-block; font-size: 12px; font-family: Helvetica; color: #000000; line-height: 1.2; pointer-events: all; white-space: normal; word-wrap: normal; "><font color="#ffffff"><span style="font-size: 19px">3</span></font></div></div></div></foreignObject><text x="190" y="249" fill="#000000" font-family="Helvetica" font-size="12px" text-anchor="middle">3</text></switch></g><ellipse cx="635" cy="146.05" rx="20" ry="20" fill="#0a6ed1" stroke="#0a6ed1" pointer-events="all"/><g transform="translate(-0.5 -0.5)"><switch><foreignObject style="overflow: visible; text-align: left;" pointer-events="none" width="100%" height="100%" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility"><div xmlns="http://www.w3.org/1999/xhtml" style="display: flex; align-items: unsafe center; justify-content: unsafe center; width: 38px; height: 1px; padding-top: 146px; margin-left: 616px;"><div style="box-sizing: border-box; font-size: 0; text-align: center; "><div style="display: inline-block; font-size: 12px; font-family: Helvetica; color: #000000; line-height: 1.2; pointer-events: all; white-space: normal; word-wrap: normal; "><font color="#ffffff"><span style="font-size: 19px">5</span></font></div></div></div></foreignObject><text x="635" y="150" fill="#000000" font-family="Helvetica" font-size="12px" text-anchor="middle">5</text></switch></g><rect x="570" y="240" width="130" height="20" fill="none" stroke="none" pointer-events="all"/><g transform="translate(-0.5 -0.5)"><switch><foreignObject style="overflow: visible; text-align: left;" pointer-events="none" width="100%" height="100%" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility"><div xmlns="http://www.w3.org/1999/xhtml" style="display: flex; align-items: unsafe center; justify-content: unsafe center; width: 1px; height: 1px; padding-top: 250px; margin-left: 635px;"><div style="box-sizing: border-box; font-size: 0; text-align: center; "><div style="display: inline-block; font-size: 12px; font-family: Helvetica; color: #000000; line-height: 1.2; pointer-events: all; white-space: nowrap; ">creates infrastructure</div></div></div></foreignObject><text x="635" y="254" fill="#000000" font-family="Helvetica" font-size="12px" text-anchor="middle">creates infrastructure</text></switch></g><rect x="850" y="245.05" width="130" height="20" fill="none" stroke="none" pointer-events="all"/><g transform="translate(-0.5 -0.5)"><switch><foreignObject style="overflow: visible; text-align: left;" pointer-events="none" width="100%" height="100%" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility"><div xmlns="http://www.w3.org/1999/xhtml" style="display: flex; align-items: unsafe center; justify-content: unsafe center; width: 1px; height: 1px; padding-top: 255px; margin-left: 915px;"><div style="box-sizing: border-box; font-size: 0; text-align: center; "><div style="display: inline-block; font-size: 12px; font-family: Helvetica; color: #000000; line-height: 1.2; pointer-events: all; white-space: nowrap; ">watches subscription</div></div></div></foreignObject><text x="915" y="259" fill="#000000" font-family="Helvetica" font-size="12px" text-anchor="middle">watches subscription</text></switch></g><rect x="860" y="125.05" width="110" height="20" fill="none" stroke="none" pointer-events="all"/><g transform="translate(-0.5 -0.5)"><switch><foreignObject style="overflow: visible; text-align: left;" pointer-events="none" width="100%" height="100%" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility"><div xmlns="http://www.w3.org/1999/xhtml" style="display: flex; align-items: unsafe center; justify-content: unsafe center; width: 1px; height: 1px; padding-top: 135px; margin-left: 915px;"><div style="box-sizing: border-box; font-size: 0; text-align: center; "><div style="display: inline-block; font-size: 12px; font-family: Helvetica; color: #000000; line-height: 1.2; pointer-events: all; white-space: nowrap; ">dispatches event</div></div></div></foreignObject><text x="915" y="139" fill="#000000" font-family="Helvetica" font-size="12px" text-anchor="middle">dispatches event</text></switch></g><path d="M 380 205 L 410 205 Q 420 205 430 204.88 L 453.63 204.58" fill="none" stroke="#666666" stroke-miterlimit="10" pointer-events="stroke"/><path d="M 458.88 204.51 L 451.93 208.1 L 453.63 204.58 L 451.84 201.1 Z" fill="#666666" stroke="#666666" stroke-miterlimit="10" pointer-events="all"/><path d="M 699 220 L 591.37 220" fill="none" stroke="#666666" stroke-miterlimit="10" pointer-events="stroke"/><path d="M 586.12 220 L 593.12 216.5 L 591.37 220 L 593.12 223.5 Z" fill="#666666" stroke="#666666" stroke-miterlimit="10" pointer-events="all"/><path d="M 587.37 199.83 L 633.83 199.83 Q 643.83 199.83 653.83 199.68 L 693.63 199.09" fill="none" stroke="#666666" stroke-miterlimit="10" pointer-events="stroke"/><path d="M 698.88 199.02 L 691.93 202.62 L 693.63 199.09 L 691.83 195.62 Z" fill="#666666" stroke="#666666" stroke-miterlimit="10" pointer-events="all"/><ellipse cx="910" cy="105.05" rx="20" ry="20" fill="#0a6ed1" stroke="#0a6ed1" pointer-events="all"/><g transform="translate(-0.5 -0.5)"><switch><foreignObject style="overflow: visible; text-align: left;" pointer-events="none" width="100%" height="100%" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility"><div xmlns="http://www.w3.org/1999/xhtml" style="display: flex; align-items: unsafe center; justify-content: unsafe center; width: 38px; height: 1px; padding-top: 105px; margin-left: 891px;"><div style="box-sizing: border-box; font-size: 0; text-align: center; "><div style="display: inline-block; font-size: 12px; font-family: Helvetica; color: #000000; line-height: 1.2; pointer-events: all; white-space: normal; word-wrap: normal; "><font color="#ffffff"><span style="font-size: 19px">6</span></font></div></div></div></foreignObject><text x="910" y="109" fill="#000000" font-family="Helvetica" font-size="12px" text-anchor="middle">6</text></switch></g><rect x="370" y="280" width="64.53" height="384.58" rx="9.03" ry="9.03" fill="#f0f6ff" stroke="none" transform="rotate(-90,402.26,472.29)" pointer-events="all"/><rect x="220" y="445.79" width="113" height="20" fill="none" stroke="none" pointer-events="all"/><g transform="translate(-0.5 -0.5)"><switch><foreignObject style="overflow: visible; text-align: left;" pointer-events="none" width="100%" height="100%" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility"><div xmlns="http://www.w3.org/1999/xhtml" style="display: flex; align-items: unsafe center; justify-content: unsafe center; width: 111px; height: 1px; padding-top: 456px; margin-left: 221px;"><div style="box-sizing: border-box; font-size: 0; text-align: center; "><div style="display: inline-block; font-size: 12px; font-family: Helvetica; color: #000000; line-height: 1.2; pointer-events: all; white-space: normal; word-wrap: normal; "><font style="font-size: 15px"><b>Reference key:</b></font></div></div></div></foreignObject><text x="277" y="459" fill="#000000" font-family="Helvetica" font-size="12px" text-anchor="middle">Reference key:</text></switch></g><rect x="292.5" y="465.79" width="306" height="33" fill="none" stroke="none" pointer-events="all"/><g transform="translate(-0.5 -0.5)"><switch><foreignObject style="overflow: visible; text-align: left;" pointer-events="none" width="100%" height="100%" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility"><div xmlns="http://www.w3.org/1999/xhtml" style="display: flex; align-items: unsafe center; justify-content: unsafe flex-start; width: 304px; height: 1px; padding-top: 482px; margin-left: 295px;"><div style="box-sizing: border-box; font-size: 0; text-align: left; "><div style="display: inline-block; font-size: 12px; font-family: Helvetica; color: #000000; line-height: 1.2; pointer-events: all; white-space: normal; word-wrap: normal; "><div><span>Dashed connectors represent background processes</span></div></div></div></div></foreignObject><text x="295" y="486" fill="#000000" font-family="Helvetica" font-size="12px">Dashed connectors represent background processes</text></switch></g><path d="M 221 482.79 L 275.63 482.79" fill="none" stroke="#666666" stroke-miterlimit="10" stroke-dasharray="3 3" pointer-events="stroke"/><path d="M 280.88 482.79 L 273.88 486.29 L 275.63 482.79 L 273.88 479.29 Z" fill="#666666" stroke="#666666" stroke-miterlimit="10" pointer-events="all"/></g><switch><g requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility"/><a transform="translate(0,-5)" xlink:href="https://www.diagrams.net/doc/faq/svg-export-text-problems" target="_blank"><text text-anchor="middle" font-size="10px" x="50%" y="100%">Viewer does not support full SVG 1.1</text></a></switch></svg>
```

Please change to the new diagram style as indicated in the guidelines https://kyma-project.io/community/guidelines/content-guidelines/08-diagrams#examples
`